### PR TITLE
feature(460483): Remove unused fields from persistence 

### PR DESCRIPTION
**Pull request by mokhld**
[Story Link](https://dev.azure.com/defragovuk/DEFRA-CDP/_boards/board/t/DXT%20Team/Stories?System.IterationPath=%40currentIteration&workitem=460483)

This PR removes unneeded fields from persistence and increases test coverage of the `ingestFile()` service method.
Also shifts the tests from `test()` to `it()` as we're moving away from that.

### CODE REVIEW

**Review by davidjamesstone — APPROVED** [review]
Nice one @mokhld 

That small import `S3FileIdentifier` fix and the ESlint warning then LGTM :+1:

File: src/api/files/service.test.js [hunk 1]
```diff
@@ -78,24 +78,21 @@ describe('Files service', () => {
...
+    fileStatus: 'complete'
   }
 
   describe('ingestFile', () => {
     beforeEach(() => {
       s3Mock.reset()
     })
 
-    test('should upload the file in the payload', async () => {
+    it('should upload the file in the payload', async () => {
```
Comment (colinrotherham) [inline src/api/files/service.test.js:95]: 💪

File: src/api/files/service.js [hunk 2]
```diff
@@ -441,5 +442,5 @@ export async function checkFileStatus(fileId) {
 /**
  * @import { SubmitPayload, SubmitRecordset } from '@defra/forms-model'
  * @import { Input, Callback } from 'csv-stringify'
- * @import { FileUploadStatus, UploadPayload } from '~/src/api/types.js'
+ * @import { FormFileUploadStatus, S3FileIdentifier, UploadPayload } from '~/src/api/types.js'
```
Comment (davidjamesstone) [inline src/api/files/service.js:445]: Unused `S3FileIdentifier`

**Review by davidjamesstone — APPROVED** [review]

File: src/api/files/service.js (hunk 2 above)
Comment (mokhld) [reply src/api/files/service.js]: I actually did this but forgot the push 😭

**Pull request merged**